js/algorithms3.js

```diff
@@ -0,0 +1,12 @@
+function bubbleSort(array) {
+    for (let i = 0; i < array.length - 1; i++) {
+        for (let j = 0; j < array.length - 1-i; j++) {
+            if (array[j] > array[j + 1]) {
+               array[j],array[j+1]=array[j+1],array[j];
```

Comment:
uu i didn't know you can actually do the swap this way
LGTM

Reply:
btw there is a way to optimize this

Reply:
I've tried different ways, but for now, this one is the fastest way I found

Reply:
the question is the clue, you can check the optimized solution here but try first to think about it for 5 min

https://www.geeksforgeeks.org/bubble-sort-algorithms-by-using-javascript/#:~:text=Bubble%20sort%20algorithm%20is%20an,increasing%20order%20or%20decreasing%20order.